packages/api/src/utils/common.ts

```diff
@@ -1,6 +1,16 @@
+export type JSONBlob = Record<string, Json>;
+
+export type Json =
+  | string
+  | number
+  | boolean
+  | null
+  | Json[]
+  | { [key: string]: Json };
```

Comment:
This can be `JSONBlob`

Reply:
Try it and see ;) It'll raise a circular definition error, at least when I did, it did.

Reply:
Ah...good point. didn't notice the circular ref. its no big deal. but I'd use Record<string, Json> instead

Reply:
Also circular, weirdly enough. It has to be a primitive literal in order to hit the circular reference type edge case.